.azure-pipelines/integrated-pipeline.yml

```
@@ -0,0 +1,102 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+name: 'Integrated Pipeline'
+variables:
+- name: AUTH_MODULE_NAME
+  value: 'Authentication'
+- name: AUTH_MODULE_PATH
+  value: 'src\Authentication\Authentication'
+- name: AUTH_MODULE_DLL_PATTERN
+  value: 'Microsoft.Graph.Authentication.dll'
+- name: MODULE_DLL_PATTERN
+  value: 'Microsoft.Graph.Authentication.dll'
+- name: BETA_MODULE_PREFIX
```

This can just be MODULE_PREFIX since we no longer modules for each service version.
```suggestion
- name: MODULE_PREFIX
```